core/server/common/src/main/java/alluxio/master/Master.java

```diff
@@ -24,7 +22,7 @@
 /**
  * This interface contains common operations for all masters.
  */
-public interface Master extends JournalCheckpointStreamable {
+public interface Master extends Iterable<JournalEntry> {
```

Comment:
I thought about this when reviewing the PR initially ... I think the ability of reuse existing utility methods and in the case of `Iterable` the for-each loop justifies its use here.